source/common/upstream/cluster_manager_impl.h

```diff
@@ -208,6 +208,7 @@ class ClusterManagerInitHelper : Logger::Loggable<Logger::Id::upstream> {
 /**
  * All cluster manager stats. @see stats_macros.h
  */
+// TODO(abeyad): do we still need the warming_clusters metric or can it be deleted?
```

Comment:
I think it's useful and could cause harm so let's leave it in.

Reply:
ACK, removed the comment